packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/list/OvirtCredentialsList.tsx

```diff
@@ -60,6 +79,13 @@ export const OvirtCredentialsList: React.FC<ListComponentProps> = ({ secret, rev
             <MaskedData />
           )}
         </div>
+        <div className="forklift-page-secret-hint-div">
+          {key === 'insecureSkipVerify' ? (
```

Comment:
why do you need this test ?
do you want to hide the hint ? why ? in what cases do you want to hide the hint ?

Reply:
Yes, just to hide this hint in case of ignoring the ca - since there is no need for a hint of how to set this ca in that case. Make the dialog less loaded with information.